src/CMakeLists.txt

```diff
@@ -94,7 +94,7 @@ include_directories (${NETCDF}/include)
 
 # Directories and libraries to include in the link step
 link_directories(${CMAKE_CURRENT_BINARY_DIR})
-link_libraries( pioc piof netcdf )
+link_libraries( pioc piof netcdf esmf )
```

Comment:
I'm surprised you had to add this. The unit tests weren't working without it?

Reply:
Goes back to this post:
https://github.com/ESCOMP/CTSM/pull/2956#issuecomment-2660211780
and the subsequent commit.